http/handler.go

```diff
@@ -1017,6 +1024,15 @@ func respondStandby(core *vault.Core, w http.ResponseWriter, reqURL *url.URL) {
 		RawQuery: reqURL.RawQuery,
 	}
 
+	// WebSockets schemas are ws or wss
+	if websocketPaths.HasPath(reqURL.Path) {
```

Comment:
I'll test that. For some reason, I thought the namespace was already removed by this point in the code, as it isn't considered in other similar code paths.

Reply:
Okay, currently, we don't handle non-root namespaces. Theoretically that is removed from the path, but `/sys` endpoints don't appear to honor namespaces from what I can tell.

I think the best approach is to have a separate PR to add proper namespace support for event subscriptions over WebSockets.

Reply:
Yeah that may need a follow-up in the ent repo. FWIW from my limited knowledge, the `/sys` mount is available but different in child namespaces. It only serves a subset of the paths available from the root namespace, so that might be why it looks a bit weird while you're looking through the code.

Reply:
I did some tests in the ent repo, and we'll definitely want to do some follow-up for the upcoming namespace work.